js/src/elements/form-behavior.ts

```diff
@@ -676,7 +684,9 @@ function setBusySubmitButton(event: Event, form: HTMLFormElement, isLoading: boo
 
 export function getLabel(input: HTMLValueElement): string {
 	const label =
-		input.labels && input.labels.length
+		input.type === 'radio'
+			? input.getAttribute('data-error-label')
+			: input.labels && input.labels.length
 			? input.labels[0].textContent
 			: input.getAttribute('aria-label');
```

Comment:
Personal preference here, I think it'd be good to try to avoid nesting ternaries for readability's sake. I'll open this up to other Atlas contributors to weigh in on how they feel, but I might recommend restructuring this to something like:

```ts
let label = '';

if (input.type === 'radio') {
    label = input.getAttribute('data-error-label');
} else if (input.labels?.length) {
    label = input.labels[0].textContent;
} else {
    label = input.getAttribute('aria-label');
}
```

Thoughts?

Reply:
Yeah.  I agree.

I had it split out to be more readable but the linting was hating me so I went path of least resistance to start.  I will bake this in and let you know when its ready for review again.

Reply:
@BenDMyers  updated